tests/test_matching.py

```diff
@@ -41,8 +41,8 @@ def test_setup_matching(self, client, test_client, helpers):
             </p>'''
         })
         print ('Homepage header set')
-        builder.set_conference_area_chairs_name('Senior_Program_Committee')
```

Comment:
is there a reason why you changed the group names?

Reply:
We have to check which is the match group so we can delete the appropriate groups when redeploying, I am assuming now the groups are always 'Area Chairs' and 'Reviewers'

Reply:
I guess I could just use is_area_chair instead